site/_layouts/site.html

```diff
@@ -0,0 +1,126 @@
+<!DOCTYPE html>
```

Comment:
What was wrong with `default.html`?

Reply:
Oh I see, you're using two different templates... Are we not planning to merge the layouts at this stage?

Reply:
Probably some parts of the layouts will be unified later, but they are quite different, right?

Reply:
Yes, I assume they will be unified, and I was hoping that would be happening this week. But if you want to phase the work such that at the moment you preserve 2 different layouts and then a future PR will come and unify them, that's fine.

Reply:
@nottrobin Yes. We want them unified. This is just a first step to get it running with 2 separate layouts, so there are no changes to the look of site or docs. Next step would be to unify them both on design and template levels.